### PR TITLE
1612472: Add back items removed during latest build

### PR DESCRIPTION
**Pull request by cnsnyder**
This adds back the equivalent of the changes from the following commits which were removed accidentally during a build.
Commits:

https://src.fedoraproject.org/rpms/subscription-manager/c/b5adf3c36513d38378c21b85e0e812a8cab67982
https://src.fedoraproject.org/rpms/subscription-manager/c/bb002d86e73ab6e401f6f03dd8eeb4bd0c1aec00
https://src.fedoraproject.org/rpms/subscription-manager/c/86ba5e1bf699e4ca2a7b8e2c157677a1f275f1a6
https://src.fedoraproject.org/rpms/subscription-manager/c/aa3278fa5e431516f6f771a156bb97955c4e2fc4
https://src.fedoraproject.org/rpms/subscription-manager/c/618e9703cbd8c7d9b4ded5409fb7066c8fd2ce22
https://src.fedoraproject.org/rpms/subscription-manager/c/af2ac8684eafd3d10f3d4e32077b8e7304af384b

Also note the builds in the changelog are a bit out of order as the spec file for the upstream project (in this repo) had produced new build which were not released concurrently as changes were made in the fedora spec file.

I'd propose that for any new release of subman we first merge any changes made to the spec file made in the fedora repo before creating a new build at all.

### CODE REVIEW

File: subscription-manager.spec [hunk 1]
```diff
@@ -63,6 +63,7 @@
 %global rhsm_package_name %{py_package_prefix}-subscription-manager-rhsm
 %define include_syspurpose 1
 %else
+%global python_sitearch %python2_sitearch
```
Comment (hroncok) [inline subscription-manager.spec]: You cut set __python to %__python2 instead to change all similar macros as well. At least in Fedora.

**Comment by cnsnyder** [comment]
Realized that SLES / SUSE does not have the __python2 or python2_sitearch macros. Added an additional conditional to account for this.

File: subscription-manager.spec [hunk 2]
```diff
@@ -65,6 +65,12 @@
 %else
 %global py_package_prefix python
 %global rhsm_package_name subscription-manager-rhsm
+
+%if !(0%{?suse_version})
+%global __python __python2
+%global python_sitearch %python2_sitearch
```
Comment (hroncok) [inline subscription-manager.spec]: you don't need this line if you have the above.

**Comment by cnsnyder** [comment]
@candlepin-pull-request-bot Please retest this.

**Comment by kahowell** [comment]
@cnsnyder FYI, has conflicts

**Comment by cnsnyder** [comment]
I believe this has been fixed. Waiting for tests to pass. Then to do another fedora build.

**Comment by jirihnidek** [comment]
@candlepin-pull-request-bot Please retest this.

**Comment by awood** [comment]
@candlepin-pull-request-bot retest this please

**Comment by cnsnyder** [comment]
@candlepin-pull-request-bot please retest this

**Comment by awood** [comment]
@candlepin-pull-request-bot retest this please

**Comment by cnsnyder** [comment]
@candlepin-pull-request-bot retest this please

File: subscription-manager.spec [hunk 3]
```diff
@@ -853,6 +856,14 @@ find %{buildroot} -name \*.py -exec touch -r %{SOURCE0} '{}' \;
...
 
 %{_datadir}/applications/subscription-manager-gui.desktop
+%{_datadir}/icons/hicolor/16x16/apps/*.png
+%{_datadir}/icons/hicolor/22x22/apps/*.png
+%{_datadir}/icons/hicolor/24x24/apps/*.png
+%{_datadir}/icons/hicolor/32x32/apps/*.png
+%{_datadir}/icons/hicolor/48x48/apps/*.png
+%{_datadir}/icons/hicolor/96x96/apps/*.png
+%{_datadir}/icons/hicolor/256x256/apps/*.png
+%{_datadir}/icons/hicolor/scalable/apps/*.svg
```
Comment (kahowell) [inline subscription-manager.spec]: I think the listings for icon files are extraneous based on our tests. Suspect this came in via a bad rebase. Going to `commit --amend` and check results.

**Comment by kahowell** [comment]
610d03c67..d696c1576: remove extraneous icon files (see outdated comment).

Once #1890 landed, this PR was rebased, and I think something went wrong during rebase.

**Pull request closed without merging**